### PR TITLE
Update README.md

### LINKED ISSUE #160: Pub.dev is showing <br/> tag on the Readme
Thanks for this amazing package!
There's something that I noticed while browsing the Readme.
As we can see in the attached screenshot, the `<br/>` tag is visible in the line just above the `Usage` header on the Readme section of the pub.dev.

br_tag

**I would love to create a fix for this issue! So, I request you to please provide me the opportunity.**

### PR DESCRIPTION
**Pull request by Raunakk02**
Removed the `<br/>` tag that was being shown on the pub.dev and added two spaces in place of it for the newline.

Fixes #160

### CODE REVIEW

File: intercom_flutter/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Pub](https://img.shields.io/pub/v/intercom_flutter.svg)](https://pub.dev/packages/intercom_flutter)
 ![CI](https://github.com/v3rm0n/intercom_flutter/workflows/CI/badge.svg)
 
-Flutter wrapper for Intercom [Android](https://github.com/intercom/intercom-android) and [iOS](https://github.com/intercom/intercom-ios) projects. <br/>
+Flutter wrapper for Intercom [Android](https://github.com/intercom/intercom-android) and [iOS](https://github.com/intercom/intercom-ios) projects.
```
Comment (deepak786) [inline intercom_flutter/README.md:6]: by removing the BR tag from here, Could you please check how it looks like on Github readMe?

Comment (Zazo032) [reply intercom_flutter/README.md:6]: https://github.com/Raunakk02/intercom_flutter/tree/issue%23160/intercom_flutter

Comment (deepak786) [reply intercom_flutter/README.md:6]: 👍

**Review by deepak786 — APPROVED** [review]

**Review by Zazo032 — APPROVED** [review]
Thanks!

**Pull request merged**